totalRP3/Locales/Locale.lua

```diff
@@ -1259,9 +1259,6 @@ If you wish to report %s's profile and you cannot target them you will need to o
 	REG_PLAYER_NOTES_ACCOUNT = "Common notes",
 	REG_PLAYER_NOTES_ACCOUNT_HELP = "These private notes are tied to your account and will be shared with all of your profiles.",
 
-	BINDING_NAME_TRP3_OPEN_TARGET_PROFILE = "Open target profile",
-	BINDING_NAME_TRP3_TOGGLE_CHARACTER_STATUS = "Toggle character status",
-
```

Comment:
Where did these go uwu?

Reply:
I'll leave this one to Sol to resolve cause I have no idea where these went.

It does sound like these are the keys for the context menu options that were removed when edit mode was born. Were these replaced with this? 
![image](https://user-images.githubusercontent.com/10636803/219951383-da886301-ae5e-4ea6-b5c7-45829a43255a.png)

Reply:
No, those strings are for the keybindings.